api/filter/http/gzip.proto

```diff
@@ -46,41 +39,33 @@ message Gzip {
     RLE = 3;
   }
 
-  // Allows selecting zlib's compression strategy. Strategy is directly
-  // related to the characteristics of the content which is being compressed. Most of the time
-  // "DEFAULT" will be the best choice, however there are situations which changing the strategy
-  // might produce better results. For example, Run-length encoding (RLE) is normally used when the
-  // content is known for having sequences which same data occurs many consecutive times. For more
-  // information about each strategy, please refer to Zlib manual. This field will be set to
-  // "DEFAULT" if not specified.
+  // A value used for selecting Zlib's compression strategy which is directly related to the
```

Comment:
Nit: zlib seems to refer to itself all lower case, e.g. see https://zlib.net/. Why did we switch to capitalization?